test/utils/skipLogic.js

```diff
@@ -291,3 +292,47 @@ tap.test('.skipLogic()', { autoend: true }, (t) => {
     t.end()
   })
 })
+
+tap.test('.skipLogic()', { autoend: true }, (t) => {
+  t.test('skipLogicGroupCheck(): should match one depth group appropriately', (t) => {
+    const scope = {
+      field: {
+        settings: {
+          showhide: null
+        }
+      }
+    }
+    const check = {
+      logicGate: 'and',
+      action: 'showhide',
+      group: [{
+        operand: '=',
+        value: 'yes'
+      }, {
+        operand: '=',
+        value: 'no'
+      }, {
+        operand: '=',
+        value: 'yes'
+      }]
+    }
+
+    skipLogicGroupCheck(scope, 'yes', check)
+    t.notOk(scope.field.show)
+
+    check.logicGate = 'or'
+    skipLogicGroupCheck(scope, 'yes', check)
+    t.ok(scope.field.show)
+    skipLogicGroupCheck(scope, 'no', check)
+    t.ok(scope.field.show)
```

Comment:
We are missing a test case for when 'or' comes out as false.

Reply:
see below, lines 339 to 343